EventKit/eventkit-cloud
# Replace GDAL Utils

## Description of Improvement

This PR pulls out the gdal_utils code into it's own separate package.  It adds that package to our dependencies and restores functionality using it.

## How to test

Pull down the changes, make sure to rebuild your dependencies and your docker containers.  Try running jobs, make sure to run jobs with various data providers, projections and formats selected. 

## Quality Assurance 

The following items have been updated:

:smile:
